response_operations_ui/error_handlers.py

```diff
@@ -19,8 +19,8 @@ def api_error(error):
 
 @app.errorhandler(401)
 def handle_authentication_error(error):
-    logger.error('Authentication failed')
-    flash('Authentication failed', category='failed_authentication')
+    logger.info('Authentication failed')
```

Comment:
I would not use a info level for an failing authentication. The min required level is warning

Reply:
Why?  Nothing is actually wrong here.  Someone just put in the wrong username and password and noone cares (but we log it anyway just incase we want to track odd usage patterns).  We have alerts set up for log level errors because if we error then something really has gone wrong.

Reply:
Thats why loggers provide different level of logs. When you do log anaylisys info are just information relveleant to the business. Warning and errors to trace not usual behavior , unexpected , errors , possible danger , failing operations , but DEF NOT use a info for authentication failed !!

Reply:
I'm going to jump in and say this is where the 'WARN' level is useful. Not something that would make me sit up and take notice, but useful enough to warrant distinguishing them from background noise of the INFO level.

Reply:
That actually seems reasonable.  Changed it to warn.  The only slight problem with it is there might be a splunk alert if there are a ton of these in 1 go, but that also seems like something we might want to know if we got a lot of warnings in a short time